src/Constants.java

```diff
@@ -0,0 +1,51 @@
+public class Constants {
```

Comment:
Shouldn't all these methods be static? Why is there a constructor and non-static methodss